docset/windows/defender/Set-MpPreference.md

```diff
@@ -752,6 +752,8 @@ The acceptable values for this parameter are: integers from 5 through 100, and t
 Windows Defender does not exceed the percentage of CPU usage that you specify.
 The default value is 50.
 
+Note: This is not a hard limit but rather a guidance for the scanning engine to not exceed this maximum on average.
```

Comment:
```suggestion
> [!NOTE]
> This is not a hard limit but rather a guidance for the scanning engine to not exceed this maximum on average.
```

Reply:
Are we including markdown, now?

Reply:
Hi @tseward - my suggestion was purely for the sake of consistency to align the format of the note with the format of the note further down in this topic. However, I am happy to leave it as is.